frontend/src/API/TeamEventsAPI.ts

```diff
@@ -0,0 +1,52 @@
+import { backendURL } from '../environment';
+import { Emitters } from '../utils';
+import APIWrapper from './APIWrapper';
+
+type TeamEventResponseObj = {
+  teamEvent: TeamEvent;
+  error?: string;
+};
+
+export type TeamEventAttendance = {
+  member: IdolMember;
+  hoursAttended?: number;
+  image: string;
+};
+
+export type TeamEvent = {
+  name: string;
+  date: string;
+  numCredits: string;
+  hasHours: boolean;
+  requests: TeamEventAttendance[];
+  attendees: TeamEventAttendance[];
+  uuid: string;
+};
```

Comment:
Aren't these in `common-types/index.d.ts` already?

Reply:
yes, fixed!